outputs.tf

```diff
@@ -17,3 +17,8 @@ output "task_role_arn" {
   description = "ECS Task role ARN"
   value       = "${module.ecs_alb_service_task.task_role_arn}"
 }
+
+output "ecs_sg_id" {
```

Comment:
Let's keep the output consistent. Rename to `service_security_group_id`

Reply:
ahh yeah, from before...